stdlib/public/core/Arrays.swift.gyb

```diff
@@ -1412,6 +1416,7 @@ extension Array {
   ///
   /// - Complexity: O(`self.count`) if the array is bridged,
   ///   otherwise O(1).
+  @discardableResult
   public mutating func popLast() -> Element? {
```

Comment:
Actually, I think I see why the `pop*` functions should warn on unused results. Unlike the `remove` and `removeLast` functions, the `pop*` functions return an optional (`nil` indicating the collection was empty). This is a success/failure indicator and shouldn't be ignored.